Warning: a recent release failed

The release PR #37 is still in a pending state after several hours

```
[ERROR] 8001: com.google.events.cloud.audit.v1.AuditLog: Class com.google.events.cloud.audit.v1.AuditLog removed
[ERROR] 7006: com.google.events.cloud.audit.v1.Auth: Return type of method 'public java.util.Map getClaims()' has been changed to com.google.events.cloud.audit.v1.Claims
[ERROR] 7005: com.google.events.cloud.audit.v1.Auth: Parameter 1 of 'public void setClaims(java.util.Map)' has changed its type to com.google.events.cloud.audit.v1.Claims
[ERROR] 7006: com.google.events.cloud.audit.v1.AuthenticationInfo: Return type of method 'public java.util.Map getThirdPartyPrincipal()' has been changed to com.google.events.cloud.audit.v1.AuthenticationInfoThirdPartyPrincipal
[ERROR] 7005: com.google.events.cloud.audit.v1.AuthenticationInfo: Parameter 1 of 'public void setThirdPartyPrincipal(java.util.Map)' has changed its type to com.google.events.cloud.audit.v1.AuthenticationInfoThirdPartyPrincipal
[ERROR] 7006: com.google.events.cloud.audit.v1.AuthorizationInfo: Return type of method 'public com.google.events.cloud.audit.v1.Resource getResourceAttributes()' has been changed to com.google.events.cloud.audit.v1.ResourceAttributes
[ERROR] 7005: com.google.events.cloud.audit.v1.AuthorizationInfo: Parameter 1 of 'public void setResourceAttributes(com.google.events.cloud.audit.v1.Resource)' has changed its type to com.google.events.cloud.audit.v1.ResourceAttributes
[ERROR] 7006: com.google.events.cloud.audit.v1.LogEntryData: Return type of method 'public com.google.events.cloud.audit.v1.LogEntryOperation getOperation()' has been changed to com.google.events.cloud.audit.v1.Operation
[ERROR] 7006: com.google.events.cloud.audit.v1.LogEntryData: Return type of method 'public com.google.events.cloud.audit.v1.AuditLog getProtoPayload()' has been changed to com.google.events.cloud.audit.v1.ProtoPayload
[ERROR] 7006: com.google.events.cloud.audit.v1.LogEntryData: Return type of method 'public java.lang.String getReceiveTimestamp()' has been changed to java.time.OffsetDateTime
[ERROR] 7006: com.google.events.cloud.audit.v1.LogEntryData: Return type of method 'public com.google.events.cloud.audit.v1.MonitoredResource getResource()' has been changed to com.google.events.cloud.audit.v1.Resource
[ERROR] 7006: com.google.events.cloud.audit.v1.LogEntryData: Return type of method 'public java.lang.String getSeverity()' has been changed to com.google.events.cloud.audit.v1.NumResponseItems
[ERROR] 7006: com.google.events.cloud.audit.v1.LogEntryData: Return type of method 'public java.lang.String getTimestamp()' has been changed to java.time.OffsetDateTime
[ERROR] 7005: com.google.events.cloud.audit.v1.LogEntryData: Parameter 1 of 'public void setOperation(com.google.events.cloud.audit.v1.LogEntryOperation)' has changed its type to com.google.events.cloud.audit.v1.Operation
[ERROR] 7005: com.google.events.cloud.audit.v1.LogEntryData: Parameter 1 of 'public void setProtoPayload(com.google.events.cloud.audit.v1.AuditLog)' has changed its type to com.google.events.cloud.audit.v1.ProtoPayload
[ERROR] 7005: com.google.events.cloud.audit.v1.LogEntryData: Parameter 1 of 'public void setReceiveTimestamp(java.lang.String)' has changed its type to java.time.OffsetDateTime
[ERROR] 7005: com.google.events.cloud.audit.v1.LogEntryData: Parameter 1 of 'public void setResource(com.google.events.cloud.audit.v1.MonitoredResource)' has changed its type to com.google.events.cloud.audit.v1.Resource
[ERROR] 7005: com.google.events.cloud.audit.v1.LogEntryData: Parameter 1 of 'public void setSeverity(java.lang.String)' has changed its type to com.google.events.cloud.audit.v1.NumResponseItems
[ERROR] 7005: com.google.events.cloud.audit.v1.LogEntryData: Parameter 1 of 'public void setTimestamp(java.lang.String)' has changed its type to java.time.OffsetDateTime
[ERROR] 8001: com.google.events.cloud.audit.v1.LogEntryOperation: Class com.google.events.cloud.audit.v1.LogEntryOperation removed
[ERROR] 8001: com.google.events.cloud.audit.v1.MonitoredResource: Class com.google.events.cloud.audit.v1.MonitoredResource removed
[ERROR] 8001: com.google.events.cloud.audit.v1.Peer: Class com.google.events.cloud.audit.v1.Peer removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public com.google.events.cloud.audit.v1.Auth getAuth()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public java.util.Map getHeaders()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public java.lang.String getHost()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public java.lang.String getID()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public java.lang.String getMethod()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public java.lang.String getPath()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public java.lang.String getProtocol()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public java.lang.String getQuery()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public java.lang.String getReason()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public java.lang.String getScheme()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public java.lang.Long getSize()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public java.lang.String getTime()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public void setAuth(com.google.events.cloud.audit.v1.Auth)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public void setHeaders(java.util.Map)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public void setHost(java.lang.String)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public void setID(java.lang.String)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public void setMethod(java.lang.String)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public void setPath(java.lang.String)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public void setProtocol(java.lang.String)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public void setQuery(java.lang.String)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public void setReason(java.lang.String)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public void setScheme(java.lang.String)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public void setSize(java.lang.Long)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Request: Method 'public void setTime(java.lang.String)' has been removed
[ERROR] 7006: com.google.events.cloud.audit.v1.RequestMetadata: Return type of method 'public com.google.events.cloud.audit.v1.Peer getDestinationAttributes()' has been changed to com.google.events.cloud.audit.v1.DestinationAttributes
[ERROR] 7006: com.google.events.cloud.audit.v1.RequestMetadata: Return type of method 'public com.google.events.cloud.audit.v1.Request getRequestAttributes()' has been changed to com.google.events.cloud.audit.v1.RequestAttributes
[ERROR] 7005: com.google.events.cloud.audit.v1.RequestMetadata: Parameter 1 of 'public void setDestinationAttributes(com.google.events.cloud.audit.v1.Peer)' has changed its type to com.google.events.cloud.audit.v1.DestinationAttributes
[ERROR] 7005: com.google.events.cloud.audit.v1.RequestMetadata: Parameter 1 of 'public void setRequestAttributes(com.google.events.cloud.audit.v1.Request)' has changed its type to com.google.events.cloud.audit.v1.RequestAttributes
[ERROR] 7002: com.google.events.cloud.audit.v1.Resource: Method 'public java.lang.String getName()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Resource: Method 'public java.lang.String getService()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Resource: Method 'public void setName(java.lang.String)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.Resource: Method 'public void setService(java.lang.String)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.ServiceAccountDelegationInfo: Method 'public java.lang.String getPrincipalEmail()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.ServiceAccountDelegationInfo: Method 'public java.util.Map getServiceMetadata()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.ServiceAccountDelegationInfo: Method 'public java.util.Map getThirdPartyClaims()' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.ServiceAccountDelegationInfo: Method 'public void setPrincipalEmail(java.lang.String)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.ServiceAccountDelegationInfo: Method 'public void setServiceMetadata(java.util.Map)' has been removed
[ERROR] 7002: com.google.events.cloud.audit.v1.ServiceAccountDelegationInfo: Method 'public void setThirdPartyClaims(java.util.Map)' has been removed
[ERROR] 7006: com.google.events.cloud.audit.v1.Status: Return type of method 'public java.lang.Object getDetails()' has been changed to com.google.events.cloud.audit.v1.Detail[]
[ERROR] 7005: com.google.events.cloud.audit.v1.Status: Parameter 1 of 'public void setDetails(java.lang.Object)' has changed its type to com.google.events.cloud.audit.v1.Detail[]
[ERROR] 7006: com.google.events.cloud.pubsub.v1.MessagePublishedData: Return type of method 'public com.google.events.cloud.pubsub.v1.PubsubMessage getMessage()' has been changed to com.google.events.cloud.pubsub.v1.Message
[ERROR] 7005: com.google.events.cloud.pubsub.v1.MessagePublishedData: Parameter 1 of 'public void setMessage(com.google.events.cloud.pubsub.v1.PubsubMessage)' has changed its type to com.google.events.cloud.pubsub.v1.Message
[ERROR] 8001: com.google.events.cloud.pubsub.v1.PubsubMessage: Class com.google.events.cloud.pubsub.v1.PubsubMessage removed
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  23.180 s
[INFO] Finished at: 2020-11-13T18:21:52Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:clirr-maven-plugin:2.8:check (default) on project google-cloudevent-types: There were 65 errors. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

--- a/.github/CONTRIBUTING.md
+++ b/.github/CONTRIBUTING.md
@@ -17,15 +17,22 @@ Then, to generate, run the ./gen.sh file in the java repo. This will generate th
 ./tools/gen.sh
 ```
 
-Observe the new files in `google-cloudevent-types/src/`.
+Observe the new files in `src/`.
 
 ## Test
   
-Run tests in the `google-cloudevent-types` :
+Run tests:
 
 ```sh
-cd google-cloudevent-types
 mvn test
+```
+
+## Prepublish Checklist
+
+Install the library to ensure it builds correctly:
+
+```
+mvn install -Dclirr.skip
 ```
 
 ## How to Contribute

--- a/.kokoro/release/stage.sh
+++ b/.kokoro/release/stage.sh
@@ -29,6 +29,7 @@ create_settings_xml_file "settings.xml"
 mvn clean install deploy -B \
   --settings ${MAVEN_SETTINGS_FILE} \
   -DskipTests=true \
+  -Dclirr.skip \
   -DperformRelease=true \
   -Dgpg.executable=gpg \
   -Dgpg.passphrase=${GPG_PASSPHRASE} \

--- a/.kokoro/release/stage.sh
+++ b/.kokoro/release/stage.sh
@@ -29,7 +29,7 @@ create_settings_xml_file "settings.xml"
 mvn clean install deploy -B \
   --settings ${MAVEN_SETTINGS_FILE} \
   -DskipTests=true \
-  -Dclirr.skip \
+  -Dclirr.skip \ # Ignore running clirr for breaking changes. See #43
   -DperformRelease=true \
   -Dgpg.executable=gpg \
   -Dgpg.passphrase=${GPG_PASSPHRASE} \

